server/src/models/results/loginResult.ts

```diff
@@ -0,0 +1,12 @@
+import { SocoboUser } from "./../database/socoboUser";
```

Comment:
Okay, this result goes to the outside. We could change this to your suggestions inside the rename issue.